src/app/dashboard/views/DiscoveriesList/DiscoveriesFilterAccordion/DiscoveriesFilterAccordion.tsx

```diff
@@ -12,6 +12,10 @@ type Props = {
   setSearchText: (searchText: string) => void;
 };
 
+export enum Labels {
+  AccordionLabel = "Filter discoveries",
```

Comment:
I'm of the opinion it's helpful to make the name reflect the actual value (e.g. `FilterDiscoveries` instead of `AccordionLabel`) where that makes sense. 

This - arguably - makes for better readability as it doesn't require you to jump through sections of the code as much as you're reading it to understand what will be displayed.